youtube/reviews.md

```diff
@@ -29,11 +29,11 @@ https://www.youtube.com/eddiejaoude?sub_confirmation=1
 
 Follow on other socials for behind the scenes footage, join discord to continue the conversation...
 
-http://eddiejaoude.io 
+https://eddiejaoude.io
 https://github.com/eddiejaoude
-http://youtube.com/eddiejaoude
-http://twitter.com/eddiejaoude
-http://instagram.com/eddiejaoude
-http://linkedin.com/in/eddiejaoude
+https://youtube.com/eddiejaoude
+https://twitter.com/eddiejaoude
+https://instagram.com/eddiejaoude
+https://linkedin.com/in/eddiejaoude
 https://discord.com/invite/jZQs6Wu
```

Comment:
Maybe change the discord link in the same PR and it'll be done by merging one PR. I'll close mine.